upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -68,3 +68,7 @@ There you can find links to upgrade notes for other versions too.
 
 - fix displaying '+' sign in product filter ([#2023](https://github.com/shopsys/shopsys/pull/2023))
     - see #project-base-diff to update your project
+
+- remove unnecessary default values for ENV variables ([#2040](https://github.com/shopsys/shopsys/pull/2040))
```

Comment:
I would change the phrasing, so it's clear it's recommended to change it from setting it in `parameters.yaml` file, but it's not necessary (application will work the same way)

Reply:
`remove unnecessary` sounds like an optional to me and do not have any better idea - any suggestions?

Reply:
not really. I thought about it with a clear head and I agree it's ok